backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -958,4 +958,79 @@ databaseChangeLog:
               - column:
                   name: date_tested_backdate
                   type: DATETIME
-                  remarks: timestamp to store a user correction to when the test was ordered
\ No newline at end of file
+                  remarks: timestamp to store a user correction to when the test was ordered
+  - changeSet:
+      id: add-data-hub-upload
+      author: tnielsen2@cdc.gov
+      comment: Add data_hub_upload table used to the automated results uploader.
+      changes:
+        - sql:
+            remarks: Create the enumerations needed for DataHubUpload.
+            sql: |
+              CREATE TYPE ${database.defaultSchemaName}.DATA_HUB_UPLOAD_STATUS as ENUM('IN_PROGRESS', 'SUCCESS', 'FAIL');
+            rollback:
+              sql: |
+                DROP TYPE ${database.defaultSchemaName}.DATA_HUB_UPLOAD_STATUS;
+        - createTable:
+            tableName: data_hub_upload
+            remarks: Track uploads and results of test_events to the data-hub
+            columns:
+              - column:
+                  name: internal_id
+                  type: *idtype
+                  remarks: The internal database identifier for this entity.
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: job_status
+                  type: ${database.defaultSchemaName}.DATA_HUB_UPLOAD_STATUS
+                  remarks: State of this upload. Enumerated value
```

Comment:
Ridiculous nitpick: some of the column comments end with a period, others do not. I prefer ending with a period, which is admittedly something of a funny affectation.